db/postgres.go

```diff
@@ -212,3 +213,41 @@ func NewPostgreSQL(uri, schema string) (PostgreSQL, error) {
 	}
 	return p, nil
 }
+
+func (p *PostgreSQL) ExtraIndexes(idxs []string) error {
+	c := 0
+	for _, idx := range idxs {
```

Comment:
Pq criar `c` ao invés de usar o retorno do `range`?

```suggestion
	for c, idx := range idxs {
```